src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenCheckedTests.cs

```diff
@@ -2367,5 +2367,391 @@ .locals init (int V_0)
+            CheckedConversionInExpressionTree_Implicit("short", "double", ConvertMethod.Convert);
+
+            // ushort
+            CheckedConversionInExpressionTree_Implicit("ushort", "ushort", ConvertMethod.None);
+            CheckedConversionInExpressionTree_Implicit("ushort", "int", ConvertMethod.ConvertChecked);
+            CheckedConversionInExpressionTree_Implicit("ushort", "uint", ConvertMethod.ConvertChecked);
+            CheckedConversionInExpressionTree_Implicit("ushort", "long", ConvertMethod.ConvertChecked);
+            CheckedConversionInExpressionTree_Implicit("ushort", "ulong", ConvertMethod.ConvertChecked);
+            CheckedConversionInExpressionTree_Implicit("ushort", "decimal", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("ushort", "float", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("ushort", "double", ConvertMethod.Convert);
+
+            // int
+            CheckedConversionInExpressionTree_Implicit("int", "int", ConvertMethod.None);
+            CheckedConversionInExpressionTree_Implicit("int", "long", ConvertMethod.ConvertChecked);
+            CheckedConversionInExpressionTree_Implicit("int", "decimal", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("int", "float", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("int", "double", ConvertMethod.Convert);
+
+            // uint
+            CheckedConversionInExpressionTree_Implicit("uint", "uint", ConvertMethod.None);
+            CheckedConversionInExpressionTree_Implicit("uint", "long", ConvertMethod.ConvertChecked);
+            CheckedConversionInExpressionTree_Implicit("uint", "ulong", ConvertMethod.ConvertChecked);
+            CheckedConversionInExpressionTree_Implicit("uint", "decimal", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("uint", "float", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("uint", "double", ConvertMethod.Convert);
+
+            // long
+            CheckedConversionInExpressionTree_Implicit("long", "long", ConvertMethod.None);
+            CheckedConversionInExpressionTree_Implicit("long", "decimal", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("long", "float", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("long", "double", ConvertMethod.Convert);
+
+            // ulong
+            CheckedConversionInExpressionTree_Implicit("ulong", "ulong", ConvertMethod.None);
+            CheckedConversionInExpressionTree_Implicit("ulong", "decimal", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("ulong", "float", ConvertMethod.Convert);
+            CheckedConversionInExpressionTree_Implicit("ulong", "double", ConvertMethod.Convert);
+
+            // decimal
```

Comment:
Sorry, there is something else I didn't understand here. I expected to see test for "decimal" to " "float" and "decimal" to "double", to complete the testing of half of the matrix.
On a related note, I didn't get why only testing half of the matrix. For instance, "decimal" to "ulong".